app/src/main/java/org/wikipedia/analytics/eventplatform/EditAttemptStepEvent.kt

```diff
@@ -17,29 +18,30 @@ class EditAttemptStepEvent(private val event: EditAttemptStepInteractionEvent) :
         private const val INTEGRATION_ID = "app-android"
         private val PLATFORM = WikipediaApp.getInstance().getString(R.string.device_type).lowercase()
 
-        fun logInit(wikiCode: String) {
-            submitEditAttemptEvent(ActionType.INIT, wikiCode)
+        fun logInit(pageTitle: PageTitle) {
+            submitEditAttemptEvent(ActionType.INIT, pageTitle)
         }
 
-        fun logSaveIntent(wikiCode: String) {
-            submitEditAttemptEvent(ActionType.SAVE_INTENT, wikiCode)
+        fun logSaveIntent(pageTitle: PageTitle) {
+            submitEditAttemptEvent(ActionType.SAVE_INTENT, pageTitle)
         }
 
-        fun logSaveAttempt(wikiCode: String) {
-            submitEditAttemptEvent(ActionType.SAVE_ATTEMPT, wikiCode)
+        fun logSaveAttempt(pageTitle: PageTitle) {
+            submitEditAttemptEvent(ActionType.SAVE_ATTEMPT, pageTitle)
         }
 
-        fun logSaveSuccess(wikiCode: String) {
-            submitEditAttemptEvent(ActionType.SAVE_SUCCESS, wikiCode)
+        fun logSaveSuccess(pageTitle: PageTitle) {
+            submitEditAttemptEvent(ActionType.SAVE_SUCCESS, pageTitle)
         }
 
-        fun logSaveFailure(wikiCode: String) {
-            submitEditAttemptEvent(ActionType.SAVE_FAILURE, wikiCode)
+        fun logSaveFailure(pageTitle: PageTitle) {
+            submitEditAttemptEvent(ActionType.SAVE_FAILURE, pageTitle)
         }
 
-        private fun submitEditAttemptEvent(action: ActionType, wikiCode: String) {
+        private fun submitEditAttemptEvent(action: ActionType, pageTitle: PageTitle) {
             EventPlatformClient.submit(EditAttemptStepEvent(EditAttemptStepInteractionEvent(action.valueString, "", OTHER.valueString,
-                INTEGRATION_ID, "", PLATFORM, 0, if (AccountUtil.isLoggedIn) AccountUtil.getUserIdForLanguage(wikiCode) else 0, 1)))
+                INTEGRATION_ID, "", PLATFORM, 0, if (AccountUtil.isLoggedIn) AccountUtil.getUserIdForLanguage(pageTitle.wikiSite.languageCode) else 0,
+                1, pageTitle.displayText, pageTitle.namespace().code())))
```

Comment:
Not sure but should we show `pageTitle.prefixText`?

Reply:
+1 to this. Using `prefixText` would be consistent with how other platforms use this schema.